sklearn/manifold/tests/test_isomap.py

```diff
@@ -114,6 +115,57 @@ def test_pipeline():
     assert .9 < clf.score(X, y)
 
 
+def test_pipeline_with_nearest_neighbors_transformer():
+    # Test chaining NearestNeighborsTransformer and Isomap with
+    # neighbors_algorithm='precomputed'
+    algorithm = 'auto'
+    n_neighbors = 10
+
+    X, _ = datasets.make_blobs(random_state=0)
+    X2, _ = datasets.make_blobs(random_state=1)
+
+    # compare the chained version and the compact version
+    est_chain = pipeline.make_pipeline(
+        neighbors.KNeighborsTransformer(
+            n_neighbors=n_neighbors + 1, algorithm=algorithm, mode='distance'),
```

Comment:
Is this `n_neighbors + 1` still needed since `KNeighborsTransformer` adds one when the mode is distance?